<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shashwat" ignored="0" name="TC-2: US - CWS - eShop - Logged in checkout" tests="1" failures="1" timestamp="2021-09-22T21:12:32 IST" time="373.32" errors="0">
  <testcase name="loggedInCustomerOrder" time="69.934" classname="framework.test.LoggedInCustomerOrder_PartyLite_Smoke_TC2">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at framework.utils.Reports.reportStep(Reports.java:72)
at framework.utils.Reports.reportStep(Reports.java:82)
at framework.test.LoggedInCustomerOrder_PartyLite_Smoke_TC2.loggedInCustomerOrder(LoggedInCustomerOrder_PartyLite_Smoke_TC2.java:112)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loggedInCustomerOrder -->
</testsuite> <!-- TC-2: US - CWS - eShop - Logged in checkout -->
